examples/withsequence-nested-result.yaml

```diff
@@ -0,0 +1,43 @@
+# This example shows how to nested withSequence loops in a Workflow.
```

Comment:
```suggestion
# This example shows how to nest withSequence loops in a Workflow.
```

sorry I had a typo in my [previous comment](https://github.com/argoproj/argo-workflows/pull/12577#discussion_r1495359070), fixed here